_layouts/default.html

```diff
@@ -11,6 +11,8 @@
     {% endif %}
     {% include open-graph.html %}
     <link rel="stylesheet" href="{{ site.cdnurl }}{{ site.baseurl }}/css/pygments-github.css">
+    <link rel="stylesheet" href="https://cdn.algorithmia.design/1.0.0-rc16/synapse.css">
```

Comment:
I think we just want our old stylesheets for now, not synapse

Reply:
Also fairly sure we can remove the synapse font files (Nunito Sans and Roboto Mono)

Reply:
Oh perfect, I assumed we'd need them since we're rendering within the context of the web client. Removed!